Ryoung27/redditClone
# added endpoints for user registration and login

added endpoints for user registration and login

closes #3 

## steps to test

pull down code
```
git fetch --all
git checkout rm-user-endpoint
```

1. make migrations
```
python manage.py makemigrations
python manage.py migrate
```
1. start server with `python manage.py runserver`
1. open up postman and post a user object to the endpoint at /auth/register/
1. you should see a return of an authorization token
1. now post the users username and password to the /auth/login/ endpoint
1. you should again see an auth token in the response

exmple user object for registration
```
{
	"username": "testuser",
	"email": "test@user.com",
	"password": "lolhacked"
}
```
